README.md

```diff
@@ -30,12 +32,14 @@ The first run will perform several actions:
 * import the osm.pbf file
 * execute the post_process SQL scripts
 
-Run `make up-d` to run the services in detached mode.
+Run `make up-d` to run the services (web and api) in detached mode.
 
 The tiles rendered by t-rex are available at `http://localhost:6767/`.
 
 The web front-end is available at `http://localhost:8082/vapour_trail.html`.
 
+The API (used by the front-end) is available at `http://localhost:5000`. The endpoint used is `/beta/<string:route_id>`.
```

Comment:
the endpoint is actually `/route/` and not `/beta/`

Reply:
nice catch :p